lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -204,7 +204,9 @@ sub login {
     _init_db($request);
     sanity_checks($database);
     $request->{login_name} = $version_info->{username};
-    if ($version_info->{status} eq 'does not exist'){
+    # $version_info->{status} isn't always defined by get_info, so useless undefined messages
+    # are generated.
+    if (defined $version_info->{status} && $version_info->{status} eq 'does not exist'){
```

Comment:
Your response on my comment here was quick (thanks) but not quite what I meant to ask for. I'll submit apr today to show you what I did mean.